google/cloud/spanner_dbapi/connection.py

```diff
@@ -140,26 +147,27 @@ def database(self):
         return self._database
 
     @property
-    def _spanner_transaction_started(self):
-        """Flag: whether transaction started at Spanner. This means that we had
-        made atleast one call to Spanner. Property client_transaction_started
-        would always be true if this is true as transaction has to start first
-        at clientside than at Spanner
+    def ddl_statements(self):
+        return self._ddl_statements
 
-        Returns:
-            bool: True if Spanner transaction started, False otherwise.
-        """
+    @property
+    def statements(self):
+        return self._statements
+
+    @property
+    def client_transaction_started(self):
```

Comment:
Have removed SPANNER_TRANSACTION_NOT_STARTED_WARNING as it was not needed and CLIENT_TRANSACTION_NOT_STARTED_WARNING now says "This method is non-operational as transaction has not been started." Let me know if it sounds ok

Reply:
SGTM, thanks!

Reply:
Sorry, reopening for one small nit-pick: Could we rename this property to `transaction_started`? Or otherwise make it internal? As mentioned above, users should not have to think about whether 'the client or Spanner' has started a transaction. For a user, it should simply be 'a transaction has been started'.